### ISSUE TITLE
General overview and detailed overview to structure document

### ISSUE THREAD

**Issue by Sarcletti** [dzhw/metadatamanagement#424]
Variable reports with many variables could become quite long. Therefore, an overview about all variables at the beginning of the document could be confusing.

One solution would be to begin with a general overview ("contents") showing all themes of the data set (using the first three characters of the name of the variable).

For an example, see this document, page 3 of the pdf:
https://www.neps-data.de/Portals/0/NEPS/Datenzentrum/Forschungsdaten/SC5/6-0-0/SC5_6-0-0_Codebook_en.pdf
If you are interested in weights, you can go from page 3 to 209 of the pdf (number 205)
There you find all weight variables and can go to all weights.

**Comment by rreitmann** [comment]
@Andy20 I do not really understand this. Is it relevant for our variable report or can we close this issue?

**Comment by Andy20** [comment]
It is important to give the variablereport some kind of userfriendly structure. Instead of having a overall table of contents right at the beginning of the document (which would be as unreadble as the rest of the document) the table should be split up in different thematic subsections without a detailed variable-overview. At the beginning of every subection there will be an detailed table of contents that only shows the variables of the following thematic section. 

Due to the fact that this is kind of an optmization of the report, we should postpone this issue.

**Comment by rreitmann** [comment]
Ok, I got it. Currently we have no topic (concept?) based grouping of variables at all.

**Comment by Andy20** [comment]
You are right. I'm not sure if the concept domain is meant to be such a theoretical topic. If so the topics could be derived from the variable names/ids  by @anfranken.

**Comment by UteH** [comment]
@svietgen when you discuss the structure of the dataset report, this discussion might be helpful, I think many of us agreed on this, generally.
